content/en/docs/tasks/run-application/run-replicated-stateful-application.md

```diff
@@ -206,15 +206,15 @@ server, and an `xtrabackup` container that acts as a
 [sidecar](https://kubernetes.io/blog/2015/06/the-distributed-system-toolkit-patterns).
 
 The `xtrabackup` sidecar looks at the cloned data files and determines if
-it's necessary to initialize MySQL replication on the slave.
+it's necessary to initialize MySQL replication on the secondary.
 If so, it waits for `mysqld` to be ready and then executes the
 `CHANGE MASTER TO` and `START SLAVE` commands with replication parameters
```

Comment:
BTW, using latest MariaDB you can use `START REPLICA` and similar commands rather than the legacy ones (untested BTW). MariaDB should be a drop in swap for MySQL.

Reply:
Replica instead of secondary seems fine and really accurately describes what is happening. I also did not know that MariaDB has added "START REPLICA" which is good to hear. I can spend some time this evening making changes to the pull request to fix this and correctly use primary as an adjective.

Should I work on the changes this evening, or would it be more prudent to wait for the Naming working group to decide on their official terminology? The only reason I submitted this morning was because last night I was working through the documentation for StatefulSets and realized that the language now seems pretty dated.

Reply:
sftim, Keep in mind that English is a language with the ability (maddening to non-native speakers of English) to turn verbs into nouns, nouns into verbs, and either into adjectives. So strictly speaking, in the context under discussion, replica, primary, secondary, master, slave etc. are **all adjectives that modify some noun**, like "node" or "host". English also has the capability (also maddening to NNSEs) to truncate a noun phrase and treat adjectives as nouns. Thus "primary host" becomes "primary" and "replica host" becomes "replica".

So in terms of clarity and translatability, “the primary MySQL host” is clearer and more accurate than “the MySQL primary”